src/io/flutter/view/FlutterView.java

```diff
@@ -182,11 +177,11 @@ private PerAppState getOrCreateStateForApp(FlutterApp app) {
   }
 
   private void addInspectorViewContent(FlutterApp app, @Nullable InspectorService inspectorService, ToolWindow toolWindow) {
+    String browserUrl = app.getConnector().getBrowserUrl();
+
+    System.setProperty("jxbrowser.license.key", "6P830J66YAO1XQRI1FNEXP8ZTHUOQTY2YUJRCN9RTARQ58QFAP63GRRH20B3P5PB7PC8");
```

Comment:
Get this key from an environment variable. And document it :)

Reply:
Ah yeah, I have tried it as an environment variable, but I'm unclear on whether we eventually want to expose the key  or if this is something that's part of the build but not in the source code. Is it okay to have the key in our .iml file?

Reply:
In general, auth keys are not included in source code, whether java or iml.

I'm not sure what the commercial auth is like. I saw some issues referring to files used for authentication, but that may have been an earlier version. In any case, this will need to be part of the build process.

Reply:
Consider doing something like reading the key from a properties file, having a demo / mock key checked in for that file, and overwriting the file with the real key as part of the build process.